rust/arrow/Cargo.toml

```diff
@@ -50,6 +50,7 @@ chrono = "0.4"
 flatbuffers = "0.6"
 hex = "0.4"
 arrow-flight = { path = "../arrow-flight", optional = true }
+clap = "2.33.0"
```

Comment:
This adds the extra dependency to every build. Adding an external test harness that compiles outside the project would be nice. I think that's better to go to [dev-dependencies]. I strongly suggest using [trybuild](https://crates.io/crates/trybuild), compiler like test harness.

Reply:
It's not clear to me how trybuild would be used in conjunction with the Archery suite, which just seems to want three CLI executables to pipe invoke.  Maybe these would be best in a separate project on the same level as `arrow`, `arrow-flight` etd? `arrow-integration` perhaps? Then I'll just import from arrow and we keep the actual library dependencies clean.

Reply:
That would be super nice actually. That will keep the library lean.